src/workspaces/dashboard/CloudInformation.ts

```diff
@@ -170,7 +170,12 @@ const GoogleCloudInformation = (props: GoogleCloudInformationProps): ReactNode =
               ]
             ),
           },
-          [storageCost?.estimate || '$ ...']
+          [
+            storageCost?.estimate || '$ ...',
+            h(InfoBox, { style: { marginLeft: '1ch' }, side: 'top' }, [
```

Comment:
We _could_ only show the info icon once the estimate is available, but I don't feel that strongly about it.

Reply:
Info icon always present:
https://github.com/user-attachments/assets/83243c48-393c-48dd-9e1f-5db10bd4b061

Info icon shows up once estimate does:
https://github.com/user-attachments/assets/7cb0fbd7-716a-438c-b7fe-2e66ed64677d

Let me know if you have a preference... I kind of like it better with the icon always there, but not super strongly.

Reply:
Either way is fine… go with keeping it always there.